### ISSUE TITLE
Use theme styles affects basic layout

### ISSUE THREAD

**Issue by draganescu** [WordPress/gutenberg#45423]
### Description

When using a row of galley block the layout is lost in the post editor if the use theme styles preference is turned off.

### Step-by-step reproduction instructions

1. Create a post
2. Insert a row block should be horizontal
3. Go to the top left dot menu of the editor
4. Click preferences
5. Turn off "use theme styles"
6. Notice the row block is now vertical

### Screenshots, screen recording, code snippet

https://user-images.githubusercontent.com/107534/199038370-96ada250-5c31-43b9-ad7c-0e620c4ab5a5.mp4

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by melvinisken** [comment]
I have a similar effect by upgrading from 6.0.3 to 6.1. All my galleries are broken. The editor view is still correct, but the acutal page is broken. I'm using Neve Theme, in case that's important. Here is the editor-view and the website view:
![editor](https://user-images.githubusercontent.com/10966010/200131587-20739505-f2f2-4008-b290-b7858b003e83.png)
![website](https://user-images.githubusercontent.com/10966010/200131588-cd8991b2-1046-418c-bd28-ce40263a245a.png)

I now downgraded to 6.0.3 and everything is fine again.

**Comment by talldan** [comment]
This still seems to be an issue. I've removed 'Needs Technical Feedback', as I don't know that any feedback is required.

This is possibly one that @andrewserong and @tellthemachines might want to be aware of.

**Comment by tellthemachines** [comment]
Yup, looks like a bug all right. Basic layout styles should be independent of whether the theme styles are applied to the editor view.

**Comment by tellthemachines** [comment]
This has since been fixed, probably due to the editor iframing work. Closing!
